cranelift/filetests/src/test_run_zkasm.rs

```diff
@@ -0,0 +1,87 @@
+//! Test command for compiling CLIF files to .zkasm, running it, and verifying their results
+//!
+//! using [RunCommand](cranelift_reader::RunCommand)s.
+
+use crate::runone::FileUpdate;
+use crate::subtest::SubTest;
+use cranelift_codegen::isa::TargetIsa;
+use cranelift_codegen::settings::Flags;
+use cranelift_codegen::{self, ir};
+use cranelift_reader::{parse_run_command, RunCommand, TestCommand, TestFile};
+use std::borrow::Cow;
+
+use crate::zkasm_codegen;
+
+struct TestRunZkasm;
+
+pub fn subtest(parsed: &TestCommand) -> anyhow::Result<Box<dyn SubTest>> {
+    assert_eq!(parsed.command, "run-zkasm");
+    if !parsed.options.is_empty() {
+        anyhow::bail!("No options allowed on {}", parsed);
+    }
+    Ok(Box::new(TestRunZkasm))
+}
+
+impl SubTest for TestRunZkasm {
+    fn name(&self) -> &'static str {
+        "run-zkasm"
+    }
+
+    fn is_mutating(&self) -> bool {
+        false
+    }
+
+    fn needs_isa(&self) -> bool {
+        false
+    }
+
+    /// Runs the entire subtest for a given target
+    /// TODO: in zkasm we don't really run test for given target, we run for zkasm target
+    fn run_target<'a>(
+        &self,
+        testfile: &TestFile,
+        _: &mut FileUpdate,
+        _: &'a str,
+        _: &'a Flags,
+        _: Option<&'a dyn TargetIsa>,
+    ) -> anyhow::Result<()> {
+        let mut zkasm_functions: Vec<Vec<String>> = Default::default();
+        let mut invocations: Vec<Vec<String>> = Default::default();
+        for (func, details) in &testfile.functions {
+            let zkasm_function = zkasm_codegen::compile_clif_function(func);
+            zkasm_functions.push(zkasm_function);
+            for comment in details.comments.iter() {
+                if let Some(command) = parse_run_command(comment.text, &func.signature)? {
+                    match command {
+                        RunCommand::Print(_) => {
+                            unreachable!()
```

Comment:
This should be a `todo!()`, right?

Reply:
Hmm, I don't know really now. I didn't see any `Print` command yet. And if we will somehow adopt cranelift tests to zkasm (for example remove `test interpret` from the header) removing print commands is normal strategy imo. But maybe `todo!()` is really better here